implementations/typescript/ockam/ockam_app/src/routes/service/ServiceCreate.svelte

```diff
@@ -9,22 +9,28 @@
   }
 </script>
 
-<div>
-  <h1>Service details</h1>
-  <div>
-    <label for="name">Service that you want to share:</label>
-    <div>
-      <input id="name" placeholder="{service}" bind:value="{service}" />
+<div class="border-b mb-4 pb-2 font-bold text-xl">Service details</div>
```

Comment:
Doesn't this go against the idea of having a separate section for styling in Svelte?
If that's possible I see the following advantages:

 - the structure of the UI is more obvious (now it's obfuscated by the styling)
 - a lot less attributes have to be repeated. For example `class="w-full px-4 bg-transparent border-none focus:outline-none text-right"` for the input fields

Reply:
> Doesn't this go against the idea of having a separate section for styling in Svelte?

Also a yes from my POV, but Tailwind's utility classes are themselves a productivity advantage, and leaning into that was a design goal I was pursuing during #5352. Having worked in backend and server-side-rendered HTML for years and full-stack development with React for a lesser number of years, I've found "semantic" HTML classes to be one of the biggest and most wasteful bikeshed topics of my entire career.

Reply:
I'll readily admit this is an appeal-to-authority, but the creator of Tailwind himself believes the `apply` feature to be a mistake/antipattern and I agree based on my own experiences.

https://twitter.com/adamwathan/status/1226511611592085504

> Confession: The `apply` feature in Tailwind basically only exists to trick people who are put off by long lists of classes into trying the framework.

> You should almost never use it 😬

> Reuse your utility-littered HTML instead.

Reply:
> the structure of the UI is more obvious

Generally the idiomatic way to preserve this property when embracing Tailwind means encapsulating the "noisy" utility classes into in-house reusable components with props, like `<ServiceTable items={...} />` or `<ActionButton>`/`CancelButton`, and so on. You build up a UI component library pretty fast and as a virtuous byproduct you get more consistent styling/flavor throughout the project rather than 8 subtly different `<button>` elements.

Reply:
> the biggest and most wasteful bikeshed topics of my entire career

That's a strong statement given the number of possible bikesheds :-).

> Generally the idiomatic way to preserve this property when embracing Tailwind means encapsulating the "noisy" utility classes into in-house reusable components with props, like <ServiceTable items={...} /> or <ActionButton>/<CancelButton>, and so on

That's what I was thinking of. Isn't that what Svelte encourages you to do as well?

Reply:
Svelte at least partially encourages you to write traditional CSS in-line within your component files. My recommendation is that we should either do that and skip adopting Tailwind, or lean into Tailwind's utility classes and skip the `<style>` element on our Svelte components in 99% of cases, but not use both concurrently.